.github/workflows/multi_numa.yml

```diff
@@ -0,0 +1,51 @@
+# Runs tests on multi-numa machine
+name: MultiNuma
+
+on: [workflow_call]
+
+permissions:
+  contents: read
+
+jobs:
+  multi_numa:
+    name: ${{matrix.os}}
+    # run only on upstream; forks will not have the HW
+    if: github.repository == 'oneapi-src/unified-memory-framework'
+
+    strategy:
+      matrix:
+        os: [ubuntu-22.04, RHEL-9]
+        build_type: [Release]
+        compiler: [{c: gcc, cxx: g++}]
+        shared_library: ['OFF']
```

Comment:
I think that variables with only one value shouldn't be included in the matrix as they don't generate new job runs.